static/js/components/PersonalTab.js

```diff
@@ -25,19 +26,27 @@ class PersonalTab extends React.Component {
   updateGender(gender) {
     const { dispatch, profile: { profileCopy } } = this.props;
```

Comment:
Sounds good, I'll change it to `unsavedProfile` which will contain:

    {
      errors: [],
      profile: {}
    }

and will be created after dispatching an action